rubix-core/src/main/java/com/qubole/rubix/core/CachingFileSystem.java

```diff
@@ -115,9 +118,10 @@ public FSDataInputStream open(Path path, int bufferSize)
 
         return new FSDataInputStream(
                 new BufferedFSInputStream(
-                        new CachingInputStream(inputStream, this, path, this.getConf(), statsMBean,
-                                               clusterManager.getSplitSize(), clusterManager.getClusterType()),
-                                                    CacheConfig.getBlockSize(getConf())));
+                        new CachingInputStream(inputStream, this, path,
+                                this.getConf(), statsMBean, clusterManager.getSplitSize(),
```

Comment:
avoid doing changes which are not relevant to the PR, please revert all changes which are not relevant.